docs/user-guide/metrics/index.rst

```diff
@@ -0,0 +1,365 @@
+   export SCHEMA_MANAGER_REGISTRY_URL=https://sasquatch-schema-registry.sasquatch:8081
+
+
+Your app doesn't use Kafka
+--------------------------
+
+If your app won't use Kafka for anything except publishing metrics, there is another config helper, `~safir.metrics.MetricsConfigurationWithKafka` that will construct an ``EventManager`` and all of its Kafka dependencies:
+
+
+.. code-block:: python
+
+   from safir.metrics import EventManager, MetricsConfiguration
+
+   config = MetricsConfigurationWithKafka()
+   manager = config.make_manager()
+
+Your app uses Kafka
+-------------------
+
+If your app uses Kafka for things other than metrics publishing (maybe it's a `FastStream`_ app), you can use the :ref:`Safir Kafka connection helpers <kafka-integration>` to create clients and pass them to the `~safir.metrics.EventManager` constructor.
+
+.. note::
+
+   The ``manage_kafaka`` parameter is ``False`` here.  This means that calling `~safir.metrics.EventManager.aclose` on your `~safir.metrics.EventManager` will NOT stop the Kafka clients.
+   You are expected to do this yourself somewhere else in your app.
+
+.. code-block:: python
+
+   from safir.kafka import KafkaConnectionSettings, SchemaManagerSettings
+   from safir.metrics import EventManager, MetricsConfiguration
+
+   kafka_config = KafkaConnectionSettings()
+   schema_manager_config = SchemaManagerSettings()
+   metrics_config = MetricsConfiguration()
+
+   # You can use this in all parts of your app
+   broker = KafkaBroker(**kafka_config.to_faststream_params())
+
+   admin_client = AIOKafkaAdminClient(
+       **kafka_config.to_aiokafka_params(),
+   )
+   schema_manager = schema_manager_config.make_manager()
+
+   return EventManager(
+       app_name=metrics_config.app_name,
+       base_topic_prefix=metrics_config.topic_prefix,
+       kafka_broker=broker,
+       kafka_admin_client=admin_client,
+       schema_manager=schema_manager,
+       manage_kafka=False,
+       noop=self.metrics_events.noop,
+   )
+
+.. _FastStream: https://faststream.airt.ai
+
+.. _event-dependency:
+
+EventDependency
+===============
+
+However you instantiate your `~safir.metrics.EventManager`, the `safir.metrics.EventDependency` will help you use it in your app.
+
+.. code-block:: python
+
+   from datetime import timedelta
+
+   from pydantic import Field
+   from safir.metrics import (
+       EventMaker,
+       EventManager,
+       EventPayload,
+       EventDependency,
+   )
+
+
+   class QueryEvent(EventPayload):
+       """Information about a user-submitted query."""
+
+       type: Literal["sync"] | Literal["async"] = Field(
+           title="Query type", description="The kind of query"
+       )
+
+       duration: timedelta = Field(
+           title="Query duration", description="How long the query took to run"
+       )
+
+
+   class Events(EventMaker):
+       def initialize(manager: EventManager) -> None:
+           self.query = await manager.create_publisher("query", QueryEvent)
```

Comment:
I think we can ditch this whole section, actually.  I initially thought we needed it because the dependency pattern was slightly confusing, but:
1. It's a pattern we use everywhere
2. I don't think the repetition and prose explanation makes it less confusing

I'm gonna ditch it, unless you disagree.